src/arrayMethodSort.js

```diff
@@ -4,8 +4,31 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  const DEFAULT_COMPARE = (a, b) => {
```

Comment:
```suggestion
  const defaultCompareFn = (a, b) => {
```

will be more informative